src/widgets/breadcrumb/__tests__/__snapshots__/breadcrumb-test.js.snap

```diff
@@ -34,7 +34,7 @@ exports[`breadcrumb() render renders transformed items correctly 1`] = `
     Object {
       "templates": Object {
         "home": "Home",
-        "separator": " > ",
+        "separator": ">",
```

Comment:
The spaces were ignored.

Reply:
https://travis-ci.org/algolia/instantsearch.js/jobs/437387566#L689 snapshots for this also need to be updated then

Reply:
I guess whether they get ignored depends on what's around right?
Because I can think of cases where those spaces matter and this may change them.
```jsx
Home<span dangerouslySetInnerHTML={{ __html: " > " }} />Category => Home > Category
Home <span dangerouslySetInnerHTML={{ __html: ">" }} /> Category => Home > Category
Home<span dangerouslySetInnerHTML={{ __html: ">" }} />Category => Home>Category

```

Reply:
This is how we inject the templates: https://github.com/algolia/instantsearch.js/blob/d72edb32935a3166c146770d914203e3f80b4571/src/components/Breadcrumb/Breadcrumb.js#L34-L40

Whitespaces will get added in any case (I made sure by adding a local story).